h/static/scripts/annotator/plugin/enhancedanchoring.coffee

```diff
@@ -83,6 +89,37 @@ class Anchor
       # Kill the list if it's empty
       delete @anchoring.anchors[index] unless anchors.length
 
+  # Scroll to this anchor
+  scrollIntoView: ->
```

Comment:
Do you think this should be a method on the Anchor or would it be more useful for the Anchor to expose a commonAncestor Element or something like this? I'm just wondering because scrollIntoView is an Element API and rather than bubbling that API up to the Anchor maybe we should just expose an underlying Element.